bounded-collections/src/lib.rs

```diff
@@ -41,20 +41,41 @@ pub trait Get<T> {
 	fn get() -> T;
 }
 
+/// A trait for querying a single value from a type.
+///
+/// The value should be a constant.
```

Comment:
Do i under stand correctly that the only difference to the normal `Get` trait is that this **MUST** be implemented by a const?  
I think that should be highlighted a bit better as to how it differs from `Get`.